### PR TITLE
LRSD-6571 [CP 2.0] Fix accessibility errors on Customer Portal

### PR DESCRIPTION
**Pull request by pedroliveirav**
[LRSD-6571](https://liferay.atlassian.net/browse/LRSD-6571)

### CODE REVIEW

File: workspaces/liferay-customer-workspace/client-extensions/liferay-customer-custom-element/src/routes/customer-portal/pages/Project/Attachments/components/TicketAttachmentsTable/_ticket_attachments_table.scss [hunk 1]
```diff
@@ -35,6 +35,12 @@
 	}
 }
 
+.delete-ticket-attachment-modal {
+	.header-modal-title {
```
Comment (ryanschuhler) [inline workspaces/liferay-customer-workspace/client-extensions/liferay-customer-custom-element/src/routes/customer-portal/pages/Project/Attachments/components/TicketAttachmentsTable/_ticket_attachments_table.scss:39]: If there is only one item in the nesting you can just consolidate into one line

Comment (pedroliveirav) [reply workspaces/liferay-customer-workspace/client-extensions/liferay-customer-custom-element/src/routes/customer-portal/pages/Project/Attachments/components/TicketAttachmentsTable/_ticket_attachments_table.scss]: Done.

File: workspaces/liferay-customer-workspace/client-extensions/liferay-customer-custom-element/src/common/components/NavSegment/NavSegment.jsx [hunk 2]
```diff
@@ -48,6 +48,7 @@ const NavSegment = ({
 			>
 				<Nav.Link
 					active={index === currentIndex}
+					aria-label={`Switch to ${item.label}`}
```
[the remark this answers is not in the document]
Comment (pedroliveirav) [reply workspaces/liferay-customer-workspace/client-extensions/liferay-customer-custom-element/src/common/components/NavSegment/NavSegment.jsx]: Done.

**Comment by ryanschuhler** [comment]
looks great, one small change i made here, but nice work! https://github.com/liferay-customer-solutions/liferay-portal/pull/369

**Pull request closed without merging**